### ISSUE TITLE
[BUG] After using wtype, the keyboard layout is forgotten

### ISSUE THREAD

**Issue by vactomas** [coffebar/hyprland-per-window-layout#9]
**Describe the bug**

I use bemoji to insert emoji into the text using wtype, and it momentarily changes the layout to different one, because it essentially switches to a new keyboard. After this, the change is normally reverted and I can you the previous layout. With hyprland-per-window-layout, once I switch back and forth between two windows, it doesn't switch to the correct one.

**To Reproduce**
1. Switch to non-English US layout
2. Use wtype
3. Switch between apps (to some other and back)
4. Layout set to English US

**Expected behavior**
Proper layout.

**Linux Distro (please complete the following information):**
 - NixOS 24.05 running version 2.4 of hyprland-per-window-layout

**Additional context**
```
activelayout>>liliums-lily58,English (US)
activelayout>>liliums-lily58,Czech (QWERTY)
activelayout>>wlr_virtual_keyboard_v1,English (US)
activelayout>>wlr_virtual_keyboard_v1,English (US)
activelayout>>wlr_virtual_keyboard_v1,error
activelayout>>asus-keyboard-2,Czech (QWERTY)
activewindow>>Alacritty,Alacritty
activewindowv2>>someid
activelayout>>liliums-lily58,English (US)
activelayout>>asus-keyboard-2,English (US)
activewindow>>org.telegram.desktop
activewindowv2>>someid
activewindowv2>>someid
workspace>>2
```

**Comment by coffebar** [comment]
Ok, check 2.5 version, should be fixed now

**Comment by vactomas** [comment]
Unfortunately, the problem still persists. I am adding additional logs. That said, I don't know whether this isn't partially caused by some Hyprland behavior.

```
activewindow>>org.telegram.desktop,Saved Messages
activelayout>>liliums-lily58,Czech (QWERTY)
openlayer>>rofi
closelayer>>rofi
activewindow>>,
activewindowv2>>,
activewindow>>org.telegram.desktop,Saved Messages
activelayout>>liliums-lily58,English (US)
activelayout>>wlr_virtual_keyboard_v1,English (US)
activelayout>>wlr_virtual_keyboard_v1,English (US)
activelayout>>wlr_virtual_keyboard_v1,error
activelayout>>liliums-lily58,Czech (QWERTY)
activelayout>>hp-hp-usb-c-dock-g5-system-control,Czech (QWERTY)
activelayout>>asus-keyboard-2,Czech (QWERTY)
activewindow>>dev.deedles.Trayscale,Trayscale
activelayout>>liliums-lily58,English (US)
activelayout>>hp-hp-usb-c-dock-g5-system-control,English (US)
activelayout>>asus-keyboard-2,English (US)
activewindow>>org.telegram.desktop,Saved Messages
```

**Comment by coffebar** [comment]
I'm not sure if you have updated version. I was able to reproduce and [fix](https://github.com/coffebar/hyprland-per-window-layout/commit/002438f63a47cb9d9ca03d77a345bf811d6c5e2f) the bug based on the request "After using wtype". Now it ignores **wlr_virtual_keyboard_v*** and I can't reproduce it anymore. 

Please double-check you don't have multiple instances running, updated to latest version and help me find steps to reproduce

**Comment by coffebar** [comment]
I tested with bemoji and can confirm that when use bemoji, something switching layout to English for the current active window. I took a quick look at the bemoji repository, but didn't find confirmation of this. But, as I see logs, the only problem is switching to En after select something with bemoji. 
After that, I can switch layouts again and hyprland-per-window-layout keeps it. 
It seems that this will not be fixed.
If bemoji need English - let it have it. From the hyprland-per-window-layout side, I don't have possibility to separate this event
